### ISSUE TITLE
Turning off Sleep does not activate screen

### ISSUE THREAD

**Issue by arkoulikosta** [Blackymas/NSPanel_HA_Blueprint#583]
Not sure if its the same issue with https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/554

I use a lux sensor and the sleep modus option to turn the screen off at night.
it works grate, the thing is that when the lux increases and i want the screen to light up, my automation does indeed turn off (disables sleep) but the screen does not turn on.
If i touch the screen, it turns on and stays on as it should.
So to sum up i want the screen to light up when sleep mode is disabled.

Is it a bug, is it a new request, is there another way to do that?

Thank anyway, keep up the good work
